cpp/examples/jsonschema/writer.cpp

```diff
@@ -39,7 +39,7 @@ int main(int argc, char** argv) {
 
   mcap::McapWriter writer;
   {
-    auto options = mcap::McapWriterOptions("x-jsonschema");
+    auto options = mcap::McapWriterOptions("");
```

Comment:
I agree with @wkalt this empty string is weird. IMO this should use the options builder pattern: `mcap::WriterOptions().library("foobar")` or named struct initialization. Right now this example looks silly.

At the least we should put "my-custom-profile" or something meaningful here rather than an empty string.

Reply:
Is `my-custom-profile` meaningful? I invite someone to come up with a meaningful example that we can use in this code.

Reply:
In my mind it is one step above a required empty string /shrug :(

Reply:
I don't think we should encourage custom profiles. People will copy paste this code and think it is required. It is an optional field, and should in most cases only be used with existing well-known profiles.

I don't know C++ but surely we can have an optional option.